memberboat/__main__.py

```diff
@@ -13,31 +13,62 @@
 #   - --dry-run -- only check and print what would happen
 
 
-def get_installation_token(owner, repo):
-	integration_id = os.environ['GITHUB_APP_ID']
-	private_key_file = os.environ['GITHUB_APP_PRIVATE_KEY_FILE']
+def get_installation_token(owner, repo, integration_id, private_key):
+	private_key = private_key
```

Comment:
@rye is this an artifact from the previous implementation? Not that it hurts to keep it, but what are your thoughts on nixing it?

Reply:
Ah yes, I meant to remove that line; it should go. I can do that.

Reply:
Done in #11.